log-collector-script/linux/eks-log-collector.sh

```diff
@@ -848,6 +848,34 @@ get_containerd_info() {
   fi
 
   ok
+
+  try "collect containerd snapshotter information"
+  get_soci_snapshotter_info
+
+  ok
+}
+
+get_soci_snapshotter_info() {
+  # Check if soci snapshotter service is running
+  if ! systemctl is-active --quiet soci-snapshotter 2>/dev/null; then
+    return
+  fi
+
+  try "Collect soci snapshotter information"
+
+  # Create soci-snapshotter directory within containerd folder
+  mkdir -p "${COLLECT_DIR}"/containerd/soci-snapshotter
+
+  # Collect service status and logs
+  systemctl status soci-snapshotter > "${COLLECT_DIR}"/containerd/soci-snapshotter/soci-snapshotter-status.txt 2>&1
+  timeout 75 journalctl -u soci-snapshotter --since "${DAYS_10}" > "${COLLECT_DIR}"/containerd/soci-snapshotter/soci-snapshotter-log.txt 2>&1 || echo -e "\tTimed out, ignoring \"soci-snapshotter log output \" "
+
+  # Collect configuration files if they exist
+  if [[ -f /etc/soci-snapshotter-grpc/config.toml ]]; then
```

Comment:
Some of the `cp` calls in this file check if the file exists first, some do not. Is it necessary, or is it okay for the `cp` to just fail if the file doesn't exist?